jiant/__main__.py

```diff
@@ -0,0 +1,609 @@
+"""Train a multi-task model using AllenNLP
+
+To debug this, run with -m ipdb:
+
+    python -m ipdb main.py --config_file ...
```

Comment:
Update.

Reply:
I get a double import warning for ipdb when I run it that way. [ipdb docs](https://github.com/gotcha/ipdb) say that formulation is python 2.7 only. Are we supporting python 2.7? 3.X? I've been using 3.6 and 3.7 for the two environments I've been working in.

Reply:
Hrm—I haven't been using ipdb lately. Does anyone else here know what the recommended practice is here?

We're only aiming to support 3.6/3.7—we should also clarify that somewhere...